bindings/java/src/main/native/Makefile.am

```diff
@@ -50,6 +50,11 @@ $(JUCX_GENERATED_H_FILES): $(STAMP_FILE)
 
 # Return CPU architecure in Java format
 arch = $(if $(filter x86_64,$(shell uname -p)),amd64,$(shell uname -p))
+
+# Only add filename suffix for ARM-based jars
+suffix = $(if $(findstring aarch64,$(shell uname -p)),-aarch64)
```

Comment:
can we add filename suffix only if arch is NOT x86_64?

Reply:
done